### PR TITLE
added contributing.md

### PR DESCRIPTION
**Pull request by Pragya2056**
I worked on #8 iusse

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+1. Don't Create Pull Request to update "Readme.md" or the "CONTRIBUTING.md" File.
+
+2. Make a file for your content and make your content limited to the file only.
+
+3. Spammy Pull Requests will be marked spam and then closed.
+
+4. The duplicate or copied content is strictly prohibited.
+
+5. The topic should be under Javascript Concepts. It might be even remotely related but no other than that.
```
Comment (nirajpdn) [inline CONTRIBUTING.md:41]: @Pragya2056 , topic could be anything but under seperate folder.

**Review by nirajpdn — CHANGES_REQUESTED** [review]
Topic could be anything related to technology.😎

**Review by nirajpdn — APPROVED** [review]
LGTM !

**Comment by Pragya2056** [comment]
I have made the changes. 
Thank you!!

**Comment by nirajpdn** [comment]
> I have made the changes. Thank you!!

@Pragya2056 for the great contributions.

**Comment by Pragya2056** [comment]
@nirajpdn

**Comment by nirajpdn** [comment]
> My pleasure!! Please add the hacktoberfest accepted label on this PR. Thank you

Sure thanks for reminding.

**Pull request merged**